extra/1-radio-stations.js

```diff
@@ -25,6 +29,26 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
+function getStations() {
+
+ let stations = getAllFrequencies();
+ 
+ let availableStations = [];
+ for (let i = 0; i < stations.length; i++) {
+   if (isRadioStation(stations[i])){
```

Comment:
Maybe assign frequency[i] to a variable called something like currentStation, to make the code more understandable.